client/lib/abtest/index.js

```diff
@@ -14,10 +14,10 @@ import i18n from 'i18n-calypso';
  */
 import activeTests from 'lib/abtest/active-tests';
 import analytics from 'lib/analytics';
-import config from 'config';
 import userFactory from 'lib/user';
 import wpcom from 'lib/wp';
 import { ABTEST_LOCALSTORAGE_KEY } from 'lib/abtest/utility';
+import { getLanguageSlugs } from 'lib/i18n-utils/utils';
```

Comment:
Should be imported from `lib/i18n-utils`?

Reply:
Yeah, makes sense. I'll change the import...